Source/WebCore/dom/TrustedTypePolicyFactory.cpp

```diff
@@ -100,21 +101,59 @@ Ref<TrustedScript> TrustedTypePolicyFactory::emptyScript() const
     return TrustedScript::create(""_s);
 }
 
-String TrustedTypePolicyFactory::getAttributeType(const String& tagName, const String& attribute, const String& elementNamespace, const String& attrNamespace) const
+String TrustedTypePolicyFactory::getAttributeType(const String& tagName, const String& attributeParam, const String& elementNamespace, const String& attrNamespace) const
```

Comment:
In WebKit coding style we normally avoid abbreviations like "param" and instead use words like "parameter".